server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -209,3 +213,30 @@ func isRootModified(root terraform.Root, modifiedFiles []string) (bool, error) {
 
 	return false, nil
 }
+
+// emits metrics for past 12 weeks in less than_x_days & more_than_x_days format
+func (r *Runner) emitOpenPRAgeMetrics(ctx workflow.Context, prs []github.PullRequest, numWeeks int) {
```

Comment:
The logic for this seems to be a little complex. Is there a reason why we can't just emit a gauge metric for how many days/weeks old a PR is, and then do metrics summations ourselves to identify the number of PRs older than X amount on average?

Reply:
I'm not sure I understand, do you mean to say we should emit metric for every single PR?

Reply:
I agree with @samrabelachew here, I don't think we need to know more than or less than, we just need to know how old it is.

Reply:
Changed the implementation to emit PRs' age in weeks.